blocks/cards/cards-cirlce-image.css

```diff
@@ -0,0 +1,104 @@
+.cards.circle-image {
+    margin: 20px 0;
+
+    h3 {
+        font-family: var(--body-font-family);
+        font-size: var(--body-font-size-m);
+        font-weight: var(--font-weight-medium);
+        color: inherit;
+    }
+
+    .card {
+        padding: 25px 0;
+        display: flex;
+        gap: 10px;
```

Comment:
Change these to have less space vertically between cards,and more horizontal space between the image and the text for each card.
```
padding: 10px 0;

gap: 16px;
```